app/views/components/_taxon-list.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  heading_level ||= 'h2'
+  items ||= []
+%>
+<% if items.any? %>
+  <ol class="app-c-taxon-list">
+    <% items.each_with_index do |item, index| %>
+      <li class="app-c-taxon-list__item">
+        <% if item[:description] %><<%= heading_level %> class="app-c-taxon-list__heading"><% end %>
+        <%=
+          link_to(
+            item[:text],
+            item[:path],
```

Comment:
Should we use path or href? I seem to remember someone suggesting we try to use naming conventions that matched what they are in the DOM.

Reply:
Matches topic list component and document list component in government frontend:
https://github.com/alphagov/collections/blob/master/app/views/components/_topic-list.html.erb#L12